src/main/java/org/kohsuke/github/GHEventInfo.java

```diff
@@ -53,7 +53,7 @@ public GHEvent getType() {
             if (e.name().replace("_", "").equalsIgnoreCase(t))
                 return e;
         }
```

Comment:
Ugh, I see the problem now.  All this is to handle the completely different strings used in the REST API and actual event calls. 
https://github.com/hub4j/github-api/blob/master/src/test/resources/org/kohsuke/github/AppTest/wiremock/testEventApi/__files/events-10.json#L283

There's got to be a better way to do this.  I'll file an issue to come back to this.